packages/algob/src/lib/tx.ts

```diff
@@ -159,3 +159,58 @@ export async function transferAsset (
   });
   return await deployer.waitForConfirmation(xtx.txId);
 }
+
+export async function transferMicroAlgosLsig (
```

Comment:
Please add doc strings to the new functions

Reply:
ok